test/commands.test.js

```diff
@@ -621,7 +622,12 @@ test('[commands.operations.getMasterConfig] success', function(assert) {
     }
   });
 
-  context.oldParameters = { old: 'stale' };
+  AWS.stub('KMS', 'decrypt', function(params, data) {
+    return 'data';
+  });
+
+  context.oldParameters = { old: 'secure:staleelats' };
+  context.overrides = { templateOptions : { AWS_DEFAULT_REGION : 'us-east-1'}};
```

Comment:
by `generalizable enough for cfn-config`, I mean if it's something that we can count on as always being there.